src/main/resources/db/changelog/init/sql/insert_data.sql

```diff
@@ -0,0 +1,50 @@
+INSERT INTO role (rol_id, rol_name, rol_created, rol_modified)
+VALUES (1, 'USER', current_timestamp, current_timestamp);
+
+INSERT INTO role (rol_id, rol_name, rol_created, rol_modified)
+VALUES (2, 'ADMIN', current_timestamp, current_timestamp);
+
+INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
+VALUES (1, 'Ukraine', 'Kyiv', current_timestamp, current_timestamp);
+
+INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
+VALUES (2, 'Ukraine', 'Lviv', current_timestamp, current_timestamp);
+
+INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
+VALUES (3, 'Ukraine', 'Kharkiv', current_timestamp, current_timestamp);
+
```

Comment:
after loc_id values "3" goes  "5". Maybe  loc_id  with values 4 is missed?

Reply:
yep, added